README.md

```diff
@@ -103,6 +103,82 @@ public class StockStepDefinitions
 }
 ```
 
+# Type Conversions
+
+Arguments to Step Methods will be converted from `string` to the declared types of the Step Method parameters when possible.
+The following conversions are supported:
+
+ - `Enum` types
+ - `Union` types with no parameters
+ - `Nullable<T>` types where the inner `T` type can be converted from `string`
+ - F# Tuple types where each element can be converted from `string`
```

Comment:
:wellactually: they're not F# specific - maybe `Tuple<T>`, esp given Union above ?